apis/python/src/tiledbvcf/dataset.py

```diff
@@ -351,17 +350,12 @@ def ingest_samples(
         :param list of str sample_uris: CSV list of sample names to include in
             the count.
         :param int threads: Set the number of threads used for ingestion.
-        :param int thread_task_size: Set the max length (# columns) of an
```

Comment:
Can you note in the PR description which arguments are being deprecated so we can remember to update the docs on release?

Reply:
For the CLI tool, we call some options "legacy" and continue to support them. The python binding is not done yet, do we want to support the "legacy" options in python too?

Reply:
Yes, I think we should keep the APIs as consistent as possible to avoid confusion about what works where (especially for me 🙃).